client/blocks/login/login-form.jsx

```diff
@@ -204,7 +204,13 @@ export class LoginForm extends Component {
 		event.preventDefault();
 
 		if ( ! this.props.hasAccountTypeLoaded ) {
-			this.props.getAuthAccountType( this.state.usernameOrEmail );
+			const usernameOrEmail = document.getElementById( this.usernameOrEmail.props.id ).value;
```

Comment:
Why we need to `document.geElementById` here and it's not enough to get it from the `ref` `this.usernameOrEmail.value` ?

If we really really need to use the DOM node, maybe use [ReactDOM. findDOMNode](https://reactjs.org/docs/react-dom.html#finddomnode)

Reply:
Unfortunately the value in the ref is not populate 😣. I will see if `ReactDom` works, though.